### PR TITLE
minor doc update to fix javadoc build

### PR DESCRIPTION
**Pull request by holgerfriedrich**
(javadoc build badge is red at the moment)

### CODE REVIEW

File: bundles/org.openhab.core.config.discovery.addon.mdns/src/main/java/org/openhab/core/config/discovery/addon/mdns/MDNSAddonFinder.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public MDNSAddonFinder(@Reference MDNSClient mdnsClient) {
     /**
      * Adds the given mDNS service to the set of discovered services.
      *
-     * @param device the mDNS service to be added.
+     * @param service the mDNS service to be added.
```
Comment (holgerfriedrich) [inline bundles/org.openhab.core.config.discovery.addon.mdns/src/main/java/org/openhab/core/config/discovery/addon/mdns/MDNSAddonFinder.java:78]: @andrewfg If you like, you could suggest a description form isResolved as well... The javadoc job is just failing due to the wrong name, and will succeed also without doc for isResolved.

Comment (andrewfg) [reply bundles/org.openhab.core.config.discovery.addon.mdns/src/main/java/org/openhab/core/config/discovery/addon/mdns/MDNSAddonFinder.java:78]: Ok.

```
     * @param service the mDNS service to be added.
     * @param isResolved indicates if mDNS has fully resolved the service information. 
```

**Review by andrewfg — APPROVED** [review]
LGTM

**Review by J-N-K — APPROVED** [review]
Thanks!

**Pull request merged**